client/src/modules/Registry/pages/Student/index.tsx

```diff
@@ -267,6 +273,27 @@ export function StudentRegistry(props: Props & { courseAlias?: string }) {
                   </Row>
                 </Col>
               </Row>
+              <Row gutter={DEFAULT_ROW_GUTTER}>
+                <Col xs={24} sm={24} md={20} lg={20} xl={20} style={{ marginBottom: 16 }}>
+                  <Row>
+                    <Col xs={24} sm={24} md={24} lg={24} xl={24}>
+                      <Form.Item
+                        label={
+                          <Typography.Title level={5} style={{ marginBottom: 0 }}>
+                            EPAM E-mail
+                          </Typography.Title>
+                        }
+                        tooltip={TEXT_EPAM_EMAIL_TOOLTIP}
+                        name="contactsEpamEmail"
+                        rules={[{ pattern: epamEmailPattern, message: 'Please enter a valid EPAM email' }]}
+                        requiredMark="optional"
+                      >
+                        <Input placeholder="first_last@epam.com" />
```

Comment:
`first_last` sounds weird a little bit :D
Maybe something like `John_Doe@epam.com` would be better?

Reply:
maybe)

Reply:
fixed